src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -195,6 +204,8 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
 
                 this.NuGetPackages.AddRange(export.Deployment.NuGetPackages);
 
+                this.AdditionalFiles.AddRange(export.Configuration.AdditionalFiles);
```

Comment:
No to both. The XmlSerializer will ensure an empty List<T> is created, even if no T nodes are in the serialised data.